### ISSUE TITLE
Schematic shows wrong LED pin

### ISSUE THREAD

**Issue by terrag42** [thehookup/Alexa-Enabled-LED-Clock#2]
The schematic shows the LEDs connected to Pin D2 which corresponds to "zone 2" which is used as a "fake LED Zone to keep the ESP busy". The schematic should show the LEDs connected to D1 which corresponds to "zone 1".

**Comment by sb43201** [comment]
Agreed. The D1 pin should be used, not the D2 pin as shown in the schematic.
